src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -1151,38 +1151,10 @@ def body(self):
 
         # now attach to existing
         self.cmd('vm unmanaged-disk attach -g {} --vm-name {} -n {} --vhd {} --caching ReadOnly'.format(
-            self.resource_group, self.vm_name, disk_name, vhd_uri))
-
-        # make it fun, let us migrate the data disk to managed image
-        self.cmd('vm stop -g {} -n {}'.format(self.resource_group, self.vm_name))
-        self.cmd('vm generalize -g {} --n {}'.format(self.resource_group, self.vm_name), checks=NoneCheck())
-        self.cmd('vm show -g {} --n {}'.format(self.resource_group, self.vm_name))
-        self.cmd('vm capture -g {} -n {} --vhd-name-prefix vmtest'.format(self.resource_group, self.vm_name),
```

Comment:
Can you create a static resource-group with a lock to preserve a properly generalized image?  It just seems dangerous to essentially remove a test.  How did the test pass in the first place to be recorded initially?